src/main/scala/stdlib/Lists.scala

```diff
@@ -90,50 +90,93 @@ object Lists extends FlatSpec with Matchers with org.scalaexercises.definitions.
     b should equal(List(res0, res1, res2, res3))
   }
 
-  /** Lists have many useful utility methods:
-   */
-  def usefulMethodsLists(res0: Int, res1: List[Int], res2: List[Int], res3: List[Int]) {
+  /** Lists have many useful utility methods.
+    *
+    * `reverse` return a reversed copy of the list:
+    */
+  def reverseLists(res0: Int, res1: List[Int]) {
     val a = List(1, 3, 5, 7, 9)
 
     // get the length of the list
     a.length should equal(res0)
 
     // reverse the list
     a.reverse should equal(res1)
+  }
 
-    // map a function to double the numbers over the list
-    a.map { v ⇒
-      v * 2
-    } should equal(res2)
+  /** `filter` tests each element of the List by applying a function you supply:
+    */
+  def filterLists(res0: List[Int], res1: List[Int], res2: List[Int]) {
+    val a = List(1, 2, 3, 5, 7, 9)
 
     // filter any values divisible by 3 in the list
     a.filter { v ⇒
       v % 3 == 0
-    } should equal(res3)
+    } should equal(res0)
+
+    // Functions over lists can use _ as shorthand
+    a.filter {
+      _ % 2 == 0
+    } should equal(res1)
+
+    // Functions over lists can use () instead of {}
+    a.filter(_ % 2 != 0) should equal(res2)
```

Comment:
Just to remind me: Was the difference between `()` and `{}` introduced somewhere else already? If possible I'd not mix the two concepts here and only focus on new things related to lists. What do you think?

Reply:
Yes we had duplicate reference  for  _ as shorthand and () instead of {} in map method. I have removed duplicate code reference from map method.